### PR TITLE
Lmp 82 update

### PR DESCRIPTION
**Pull request by esajaa**
New LmP requires static UID/GID defined for added users

### CODE REVIEW

File: meta-lmp-support/files/pelion-group-table [hunk 1]
```diff
@@ -0,0 +1,8 @@
+parsec:x:1004:
```
Comment (petedyerarm) [inline meta-lmp-support/files/pelion-group-table:1]:
```suggestion
parsec:x:1206:
```

File: meta-lmp-support/files/pelion-passwd-table [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parsec:x:1004:1004::/:/sbin/nologin
```
Comment (petedyerarm) [inline meta-lmp-support/files/pelion-passwd-table:1]:
```suggestion
parsec:x:1206:1206::/:/sbin/nologin
```

File: meta-lmp-support/files/pelion-group-table (hunk 1 above)
Comment (petedyerarm) [reply meta-lmp-support/files/pelion-group-table:1]: The current values used (in 2.4.0) are 1206, and I think these need to be used going forwards. An ostree update won't take the new values into use, due to the 3 way merge functionality.

**Review by petedyerarm — CHANGES_REQUESTED** [review]

File: meta-lmp-support/files/pelion-passwd-table [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+maestro:x:1200:900::/:/sbin/nologin
+deviceos:x:1201:901::/:/sbin/nologin
+developer:x:1202:1022::/:/sbin/nologin
+support:x:1203:1023::/:/sbin/nologin
+user:x:1204:1024::/:/sbin/nologin
+developer2:x:1212:1020::/:/sbin/nologin
```
Comment (petedyerarm) [inline meta-lmp-support/files/pelion-passwd-table:7]: Is nologin corrrect for all of these users? Seems better in terms of security, but I do not know if anything relies on them.

Comment (ygoyal18) [reply meta-lmp-support/files/pelion-passwd-table:7]: @petedyerarm does this mean that only `fio` user is allowed to login? and I think by default root login is disabled.

Comment (petedyerarm) [reply meta-lmp-support/files/pelion-passwd-table:7]: That is my understanding. 
Travis indicated that some of these users should be removed, but that could be done later.

**Comment by x85446** [comment]
merging as PR checks are not working

**Pull request merged**